packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableResizer.ts

```diff
@@ -35,7 +35,7 @@ export default function createTableResizer(
 
     div.style.width = `${TABLE_RESIZER_LENGTH}px`;
     div.style.height = `${TABLE_RESIZER_LENGTH}px`;
-    document.body.appendChild(div);
+    table.insertAdjacentElement('afterend', div);
```

Comment:
do we also need to do the same change for other table resize helper elements? Actually there are 6:
1. Whole table selector
2. Whole table resizer (these 2 are visible as a gray square)
3. horizontal resizer
4. vertical resizer
5. new column inserter (a circle with "+" sign when hover above a table)
6. new row inserter (a circle with "+" sign when hover on left side (or right side when RTL) of a table)

3,4 are invisible, but it will change mouse cursor when hover
5,6 will be visible when hover.

I can see your change fixed 1 and 2, so we also need same fix for 3-6. And maybe we should have a unified function for all of them to reduce duplicated code.

Reply:
I noticed that but wanted to do that on a separate PR. I think a better approach would be to do something like Image Edit does, where the helpers are in in a span.

Reply:
It's ok to do it separately, and please open an issue to track it.

I think it should be fine to use a SPAN for all the resizer UI, although this seems like a larger change.